Add a simple script for running the tests from a standalone js shell.

sunfishcode opened the pull request.

bnjbvr commented:
Nice! Note that `equal` should also test whether the two parameters are both NaN or not (same applies to `notEqual`)

bnjbvr commented:
Also, should `fail` throw the error it creates?

sunfishcode commented:
NaN isn't equal to NaN ;-). But also, QUnit's equal and notEqual don't check for NaNs, and that's what this script is imitating.

To make it easier to test for NaNs, I added a helper function in one of the patches in #122: https://github.com/sunfishcode/ecmascript_simd/commit/754574a2a3ecfbfeef5f1a06da3441b0c22b9fe3

I didn't make ```fail``` throw because I find it convenient to see all the failures at once rather than stopping at the first one. But, it would be nice to make the script indicate whether any tests failed to the system; I've now added a patch to do that.
